docs/content/install.sh

```diff
@@ -146,6 +146,10 @@ cd $unzip_dir/*
 
 case $OS in
   'linux')
+    rclonepath="$(command -v rclone)"
+    if [ -x "$rclonepath" ] && ! [ "$rclonepath" = '/usr/local/bin' ]; then
```

Comment:
Correct me if I am wrong, but shouldn't the check be to `/usr/local/bin/rclone` instead of just `/usr/local/bin` ?

Reply:
I think you are correct...